numpy/core/tests/test_mem_overlap.py

```diff
@@ -683,6 +686,7 @@ def check_unary_fuzz(self, operation, get_out_axis_size, dtype=np.int16,
                         # Check result
                         assert_copy_equivalent(operation, [a], out=b_out, axis=axis)
 
+    @dec.slow
```

Comment:
actually this and the following ufunc test are not in 1.12 I think, so maybe we can keep them or make them faster by checking for redundant checks. They take about 10s so almost one fifths of the test time.